heroku/cli
# refactor(pg-v5): Move command pg:vacuum-stats to oclif

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001bDU0sYAG/view

This converts `pg:vacuum-stats` to oclif. No test currently exists for this command, so I've created a WI to to that: https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001phGC1YAM/view

### Testing

1) Pull down this branch
2) Run `./bin/run pg:vacuum-stats --app APP` on an app with a pg database
4) Verify the command succeeds
5) Compare with prod
